docs/next-release-notes.md

```diff
@@ -0,0 +1,517 @@
+
+### New metrics for streaming ingestion
+
+The following metrics related to streaming ingestion have been added:
+
+- `ingest/kafka/partitionLag`: Partition-wise lag between the offsets consumed by the Kafka indexing tasks and latest offsets in Kafka brokers. 
+- `ingest/kinesis/partitionLag/time`: Partition-wise lag time in milliseconds between the current message sequence number consumed by the Kinesis indexing tasks and latest sequence number in Kinesis.
+- `ingest/pause/time`: Milliseconds spent by a task in a paused state without ingesting.|dataSource, taskId| < 10 seconds.|
+
+https://github.com/apache/druid/pull/13331
+https://github.com/apache/druid/pull/13313
+
+### Backoff for HttpPostEmitter
+
+The `HttpPostEmitter` option now has a backoff. This means that there should be less noise in the logs and lower CPU usage if you use this option for logging. 
+
+https://github.com/apache/druid/pull/12102
+
+### taskActionType dimension for task/action/run/time metric
+
+The `task/action/run/time` metric for the Indexing service now includes the `taskActionType` dimension.
+
+https://github.com/apache/druid/pull/13333
+
+### DumpSegment tool for nested columns
+
+The DumpSegment tool can now be used on nested columns with the `--dump nested` option. 
+
+For more information, see [dump-segment tool](https://druid.apache.org/docs/latest/operations/dump-segment).
+
+https://github.com/apache/druid/pull/13356
+
+### Segmen loading and balancings
+
+#### cachingCost balancer strategy
+
+The `cachingCost` balancer strategy now behaves more similarly to cost strategy. When computing the cost of moving a segment to a server, the following calculations are performed:
+
+- Subtract the self cost of a segment if it is being served by the target server
+- Subtract the cost of segments that are marked to be dropped
+
+#### Segment assignment
+
+You can now use a round-robin segment strategy to speed up initial segment assignments.
+
+Set `useRoundRobinSegmentAssigment` to `true` in the Coordinator dynamic config to enable this feature.
+
+https://github.com/apache/druid/pull/13367
+
+#### Segment batch balancing
+
+Batch sampling is now the default method for sampling segments during balancing as it performs significantly better than the alternative when there is a large number of used segments in the cluster.
+
+As part of this change, the following properties have been deprecated and will be removed in future releases:
+
+- `coordinator dynamic config useBatchedSegmentSampler`
+- `coordinator dynamic config percentOfSegmentsToConsiderPerMove`
+- non-batch method of sampling segments used during coordinator duty `BalanceSegments`
+
+The unused coordinator property `druid.coordinator.loadqueuepeon.repeatDelay` has been removed.
+
+Use only `druid.coordinator.loadqueuepeon.http.repeatDelay` to configure repeat delay for the HTTP-based segment loading queue.
+
+https://github.com/apache/druid/pull/13391
+
+#### Segment discovery
```

Comment:
This is also a behavior change that we should highlight.

Reply:
As part of this highlight, you should include details on how to revert to old behavior.